src/Uno.UI/Microsoft/UI/Xaml/Controls/LayoutPanel/LayoutPanel.Properties.cs

```diff
@@ -16,7 +16,11 @@ public Brush BorderBrush
 
 		private static Brush GetBorderBrushDefaultValue() => SolidColorBrushHelper.Transparent;
 
-		[GeneratedDependencyProperty(ChangedCallback = false, Options = FrameworkPropertyMetadataOptions.ValueInheritsDataContext)]
+		[GeneratedDependencyProperty(ChangedCallback = false, Options = FrameworkPropertyMetadataOptions.ValueInheritsDataContext
+#if __WASM__
+			| FrameworkPropertyMetadataOptions.AffectsArrange
+#endif
+			)]
```

Comment:
My guess would be that it is supposed to be true on all platforms, but it would make sense to affect the arrange only when we detect that arrange changes are needed (changing the brush without the thickness should not necessarily invalidate the arrange?).

Reply:
> changing the brush without the thickness should not necessarily invalidate the arrange

Changing the brush to/from null should affect arrange. I could switch from metadata options to a manual `InvalidateArrange` call only for this case.

Reply:
I haven't found the need for the other platforms though.